.changeset/bright-zoos-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/progressbar": patch
+---
+
+Reverts the background-color property to background to support linear-gradients. Gradient support stories were added to the meter and progress bar components within their Chromatic testing previews to ensure we continue to support linear-gradients as necessary.
```

Comment:
```suggestion
Reverts the `background-color` property to `background` to support the use of CSS gradients. Gradient support stories were added to the meter and progress bar components within their Chromatic testing previews to ensure that we continue to support gradients by covering them in our visual regression tests.
```
A little adjustment to markdown format the code and not limit the type of gradients mentioned (could be radial, etc).

Reply:
Thanks for this- I honestly was struggling to word this changeset description, so this is super helpful.

Reply:
https://github.com/adobe/spectrum-css/pull/2929/commits/8062e3da72b5f3ee423092e8c5286c9b101f968d